packages/components/src/core/Button/index.stories.tsx

```diff
@@ -56,121 +62,193 @@ export const Default = {
 
 // Live Preview
 
-const placementStyles = {
-  display: "grid",
-  gridColumnGap: "10px",
-  gridRowGap: "0px",
-  gridTemplateColumns: "repeat(6, 120px)",
-  gridTemplateRows: "1fr",
+const styleLevel = {
```

Comment:
Adding new / replacing old styles to better lay out all the permutations for Button component in the UI, using CSS Grid